x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_rule_actions/test_utils.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { ActionTypeModel } from '../../../../../../../../../plugins/triggers_actions_ui/public/types';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { TypeRegistry } from '../../../../../../../../../plugins/triggers_actions_ui/public/application/type_registry';
```

Comment:
For these I would leave a comment linking to an Alerting issue requesting these types be exposed.